### ISSUE TITLE
malloc errors instead of meaningful errors

### ISSUE THREAD

**Issue by bvdputte** [sass/node-sass#1347]
The source of my issue is that if there is a scss error in the files (missing semicolon, missing closing bracket, typo in the variable name, commented out variable, etc.) it aborts gulp that is run with "gulp sass".

Error: 
![screen shot 2016-01-19 at 09 57 51](https://cloud.githubusercontent.com/assets/490505/12415235/d8724b92-be9a-11e5-8d22-df7e522f944a.png)

Used versions:
node: v5.2.0
npm: 3.3.12
"gulp": "~3.9.0",
"gulp-sass": "^2.1.1",
node-sass: 3.4.1

I've also tried with
node: v4.2.4
npm: 2.14.12
to no avail :-(

The command:
````
./node_modules/gulp-sass/node_modules/node-sass/bin/node-sass --version
```
results in:
```
node-sass	3.4.2	(Wrapper)	[JavaScript]
libsass  	3.3.2	(Sass Compiler)	[C/C++]
```

The command:
```
./node_modules/.bin/node-sass --version
```
results in:
```
node-sass	3.4.2	(Wrapper)	[JavaScript]
libsass  	3.3.2	(Sass Compiler)	[C/C++]
```

The command:
```
node -p "console.log(require('node-sass').info)
```
results in:
```
"
node-sass	3.4.2	(Wrapper)	[JavaScript]
libsass  	3.3.2	(Sass Compiler)	[C/C++]
undefined
```

Demo files for this bug can be found here: https://gist.github.com/bvdputte/1529fa195f2be2ebb04f

Imho this is the same issue as https://github.com/sass/node-sass/issues/1281

**Comment by bvdputte** [comment]
FYI: compiling this sass code with ruby sass results in this:

![screen shot 2016-01-19 at 11 09 46](https://cloud.githubusercontent.com/assets/490505/12415670/428770e6-be9d-11e5-87ed-dd603e2a6445.png)

which is much more meaningful imho.

**Comment by xzyfer** [comment]
This is fixed in the upcoming 3.5.0-beta. Subscribe to #1349 for further updates.
